data/api/client-server/definitions/public_rooms_chunk.yaml

```diff
@@ -33,7 +33,9 @@ properties:
     example: "!abcdefg:example.org"
   topic:
     type: string
-    description: The topic of the room, if any.
+    description: |-
+      The plaintext topic of the room. Omitted if no `text/plain` mimetype
```

Comment:
```suggestion
      The plain text topic of the room. Omitted if no `text/plain` mimetype
```

Reply:
I'm actually not sure which one is correct. The spec seems to use both variants in different places. Maybe a native speaker could help us confirm this?

Reply:
Makes sense. Thanks for explaining. 👍